plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplitManager.java

```diff
@@ -259,6 +263,7 @@ private List<DeltaLakeSplit> splitsForFile(
                     fileSize,
                     addFileEntry.getModificationTime(),
                     ImmutableList.of(),
+                    SplitWeight.fromProportion(Math.min(Math.max((double) fileSize / splitSize, minimumAssignedSplitWeight), 1.0)),
```

Comment:
`fileSize / splitSize` I think what we want here is the `splitSize` divided by the intended split size. 

Tweaked from the code above:
```
            long maxSplitSize;
            if (remainingInitialSplits.get() > 0 && remainingInitialSplits.getAndDecrement() > 0) {
                maxSplitSize = getMaxInitialSplitSize(session).toBytes();
            }
            else {
                maxSplitSize = getMaxSplitSize(session).toBytes();
            }
            long splitSize = Math.min(maxSplitSize, fileSize - currentOffset);

            SplitWeight weight = SplitWeight.fromProportion(Math.min(Math.max((double) splitSize / maxSplitSize, minimumAssignedSplitWeight), 1.0))
```